### ISSUE TITLE
Accessibility Issue - Getting "Invalid aria-describedby" error and other alerts on Team roles fields.

### ISSUE THREAD

**Issue by Shalini-git25** [DFE-Digital/find-information-products-services-frontend#225]
**Describe the bug**
On 'Request a new product entry' page, ran tests using 'ARC toolkit' to test accessibility issues. In results, getting "Invalid aria-describedby" error on Team roles fields (Delivery Manager, Product Manager and Senior Responsible Officer). Also alert is showing for these 3 team roles fields as "ARIA attribute value is missing" which should be fixed.

When submitted blank form to test mandatory fields and then ran tests using 'ARC toolkit' it shows "Invalid aria-describedby" error for 6 fields - Product title, Description, Team role fields and Notes.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'Request a new product entry' page.
2. Open ARC Toolkit in dev tools.
3. Select 'Run tests' to start the automated accessibility checks.
4. In Result section showing error as ""Invalid aria-describedby" and alert as "ARIA attribute value is missing".
5. Check that Team roles fields are highlighted.

**Expected behavior**
Any accessibility alerts which seems to be issue should be fixed and no errors should be present on 'Request a new product entry' page.

**Screenshots**
1. 
<img width="2240" height="1320" alt="Image" src="https://github.com/user-attachments/assets/57fe3cf6-ccde-40c9-b43d-eb378fb7fc98" />

2. 
<img width="2233" height="1334" alt="Image" src="https://github.com/user-attachments/assets/37f46380-f355-44e8-807a-a647a8a3cb6d" />

3. 
<img width="1119" height="683" alt="Image" src="https://github.com/user-attachments/assets/8e1cd6ec-117b-41e1-8bf5-491b30def245" />

4. 
<img width="1125" height="663" alt="Image" src="https://github.com/user-attachments/assets/bc48f1d4-59c2-458a-97fd-a5d663617ec0" />
